### PR TITLE
Save inotify-derived grain sizes to database

### PR DESCRIPTION
**Pull request by kentonv**
After this change:

* Storage quota can be enforced on single-machine Sandstorm. (However, the UI needs to be detangled from the Oasis payments code -- @jparyani is working on that.)
* The size of each grain is shown in the grain list. (However, preexisting grains won't show a size until they are next opened.)

### CODE REVIEW

**Comment by ocdtrekkie** [comment]
> The size of each grain is shown in the grain list. (However, preexisting grains won't show a size until they are next opened.)

I am super excited about this.

File: shell/server/core.js [hunk 1]
```diff
@@ -137,6 +137,31 @@ class SandstormCoreImpl {
       logActivity(this.grainId, null, event);
     });
   }
+
+  reportGrainSize(bytes) {
+    bytes = parseInt(bytes);  // int64s are stringified but precision isn't critical here
```
Comment (zarvox) [inline shell/server/core.js]: nit: please don't reassign method arguments.  Bind a new name, so that each variable binding always has exactly one type (and where possible, exactly one value).

(I should get ESLint up and running some time Soon™ now that we have node 4...)

File: shell/client/grain/grainlist-client.js [hunk 2]
```diff
@@ -463,6 +464,12 @@ Template.sandstormGrainTable.helpers({
     return !!(Template.instance().data.alwaysShowTableHeaders ||
               Template.instance().data.grains.length);
   },
+
+  grainSize: function () {
+    // TODO(cleanup): extract prettySize and other similar helpers from globals into a package
+    // TODO(cleanup): access Meteor.user() through db object
```
Comment (zarvox) [inline shell/client/grain/grainlist-client.js]: This second TODO seems to be unrelated to the code here
